lib/Github/Api/PullRequest/Review.php

```diff
@@ -159,11 +159,22 @@ public function submit($username, $repository, $pullRequest, $id, array $params
      * @param string $repository  the repository
      * @param int    $pullRequest the pull request number
      * @param int    $id          the review id
+     * @param string  $message    a mandatory dismissal message
      *
      * @return array|string
      */
-    public function dismiss($username, $repository, $pullRequest, $id)
+    public function dismiss($username, $repository, $pullRequest, $id, $message)
```

Comment:
I was also in doubt if this was a BC break but indeed the message is required. It could be this changed in the preview period. So this should be documented but it's a correct change. See https://developer.github.com/v3/pulls/reviews/#input-2